R/rectangle.R

```diff
@@ -107,77 +64,14 @@
 #' )
 #' df
 #'
-#' # Turn all components of metadata into columns
-#' df %>% unnest_wider(metadata)
-#'
-#' # Choose not to simplify list-cols of length-1 elements
-#' df %>% unnest_wider(metadata, simplify = FALSE)
-#' df %>% unnest_wider(metadata, simplify = list(color = FALSE))
-#'
 #' # Extract only specified components
 #' df %>% hoist(metadata,
 #'   "species",
 #'   first_film = list("films", 1L),
 #'   third_film = list("films", 3L)
 #' )
-#'
-#' df %>%
-#'   unnest_wider(metadata) %>%
-#'   unnest_longer(films)
-#'
-#' # unnest_longer() is useful when each component of the list should
-#' # form a row
-#' df <- tibble(
-#'   x = 1:3,
-#'   y = list(NULL, 1:3, 4:5)
-#' )
-#' df %>% unnest_longer(y)
-#' # Automatically creates names if widening
-#' df %>% unnest_wider(y)
-#' # But you'll usually want to provide names_sep:
-#' df %>% unnest_wider(y, names_sep = "_")
-#'
-#' # And similarly if the vectors are named
-#' df <- tibble(
-#'   x = 1:2,
-#'   y = list(c(a = 1, b = 2), c(a = 10, b = 11, c = 12))
-#' )
-#' df %>% unnest_wider(y)
-#' df %>% unnest_longer(y)
-#'
-#' # Both unnest_wider() and unnest_longer() allow you to unnest multiple
-#' # columns at once. This is particularly useful with unnest_longer(), where
-#' # unnesting sequentially would generate a cartesian product of the rows.
-#' df <- tibble(
-#'   x = 1:2,
-#'   y = list(1:2, 3:4),
-#'   z = list(5:6, 7:8)
-#' )
-#' unnest_longer(df, c(y, z))
-#' unnest_longer(unnest_longer(df, y), z)
-#'
-#' # With JSON, it is common for empty elements to be represented by `list()`
-#' # rather then their typed equivalent, like `integer()`
-#' json <- list(
-#'   list(x = 1:2, y = 1:2),
-#'   list(x = list(), y = 3:4),
-#'   list(x = 3L, y = list())
-#' )
-#' df <- tibble(json = json)
-#'
-#' # The defaults of `unnest_wider()` treat empty types (like `list()`) as `NULL`.
-#' # This chains nicely into `unnest_longer()`.
-#' wide <- unnest_wider(df, json)
-#' wide
-#'
-#' unnest_longer(wide, c(x, y))
-#'
-#' # To instead enforce strict vctrs typing rules, use `strict`
-#' wide_strict <- unnest_wider(df, json, strict = TRUE)
-#' wide_strict
-#'
-#' try(unnest_longer(wide_strict, c(x, y)))
 #' @export hoist
+#' @family rectangling
```

Comment:
<img width="488" alt="Screen Shot 2022-06-10 at 11 15 31 AM" src="https://user-images.githubusercontent.com/19150088/173096951-aacc983a-bc0e-408e-bb65-894be68f36ae.png">

FWIW this is what I see from using the family tag. I wish it showed `unnest()` but I guess that is ok

Reply:
Yeah, maybe `nest() `and `unnest()` should be separate pages? I'll take a look